products/views.py

```diff
@@ -55,10 +55,15 @@ def get(self, request, product_id):
                     'url': image.url
                     } for image in images],
                 'options' : [{
-                    'id'  : option.id,
-                    'name': option.name
+                    'option_id'                 : option.id,
+                    'name'                      : option.name,
+                    'product_option_information': [{
+                        'product_option_id'   : p_o.id,
+                        'product_option_stock': p_o.stock
+                        } for p_o in product.productoption_set.all()],
```

Comment:
```suggestion
                    'product_option_information': [{
                        'product_option_id'   : product_option.id,
                        'product_option_stock': product_option.stock
                        } for product_option in product.productoption_set.all()],
```
코드는 항상 누가 보더라도 목적과 의도를 알아보기 쉽게 작성하는 것이 좋습니다. 그러기 위해선 기본적인 컨벤션을 맞춰야하고 가능하면 네이밍을 명확히 해주는 것이 필수입니다.

Reply:
네 수정하겠습니다